.github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!---
+    Thank you for contributing to Unity.
+    To help us process this pull request we recommend that you add the following information:
+     - Summary and list of changes in the pull request,
+     - Issue(s) related to the changes made such as GitHub or Jira,
+     - Manual testing scenarios if available
+    Fields marked with (*) are required. Please don't remove the template.
+-->
+### Description (*)
```

Comment:
What do you think of having two sections: one for the reviewers, and one for users when they read the changelist.
If I made a typo in “the many fox are orange”
We could have the following in a PR:
Description:
Just fixed the word "fox" to "foxes" here
Changelist item:
Fixed typos

Another PR could read:
Description:
Just fixed the word "lazy dog" to "lazy dogs"
Changelist item:
Fixed typos --> this changelist would be merged with the first one

What do you think?

Reply:
I like this idea! Something like this?

### Description (*)

<---    Please provide a description of the changes proposed in the pull request.   Letting us know what has changed and why it needed changing will help us validate this pull request.-->

### Changelist (*)

<--- Please provide a list of technical changes to code. -->